README.md

```diff
@@ -4,26 +4,42 @@
 
 🧱 A composable solution for building reusable lists in Swift/UIKit. As easy as playing with Legos.
 
-- [Features](#features)
-- [Installation](#installation)
-    - [Swift Package Manager](#swift-package-manager)
-- [Contributing](#contributing)
-- [Authors](#authors)
-
-# Features
+- [BricksAndTiles](#bricksandtiles)
+  - [Features](#features)
+    - [UITableView](#uitableview)
+    - [UICollectionView](#uicollectionview)
+    - [UITableView + UICollectionView](#uitableview--uicollectionview)
+  - [Installation](#installation)
+      - [Swift Package Manager](#swift-package-manager)
+  - [Contributing](#contributing)
+  - [Authors](#authors)
+
+## Features
 ### UITableView
 - [x] Static cells
```

Comment:
This one could have a gif of the first screen

Reply:
Ah meaning the home screen of the Example app?

Reply:
Yes! That's a static example itself.

Reply:
Although maybe there could be an explicit example of this feature, I think the home screen is good enough for now

Reply:
<img src="https://user-images.githubusercontent.com/38720242/95614034-30680f00-0a66-11eb-942d-5612cff9231e.png" width="200 px">

Reply:
Yep, this one! What do you think?

Reply:
Or we could add a static cell for examples sections... I can help you with this...

Reply:
I'm really not sure of what's better 😅

Reply:
I really believe it would be best to give an example in line with the Song / Music example, otherwise visitors will recognize it as a demonstration use case better rather than the demo menu.

I'll see if I can modify the text to capture a gif for the README, but maybe we could create a new separate issue just to add to the Example for the long term.

Reply:
Agreed. I think that's ok to leave this PR as it is then. Later tonight I'll work on the new example and try to add its gif as well. Sounds good?

Reply:
Sounds great @mugbug ! Thank you 😄